### PR TITLE
examples(nodejs): bump to the latest 3.x version of cloudevents SDK

### PR DESCRIPTION
**Pull request by lance**
A few changes to the API were made in the recent release of the SDK. This commit updates the dependency and modifies the source to work with the new API.

## Proposed Changes 

- Update the Node.js example to use the `cloudevents` module version 3.1.0
- Modify the example code to work with the changed API
- Return the actual `CloudEvent` instance in the response body in `receiveAndReply()`

### CODE REVIEW

**Comment by markusthoemmes** [comment]
I guess we have no test automation here and i'll have to manually check this is sound? :sweat_smile:

**Comment by markusthoemmes** [comment]
Actually @csantanapr would you mind giving this a look? You're more knowledgeable on this front than I am.

The changes look good on the surface though, thanks @lance!

**Comment by lance** [comment]
@markusthoemmas new release of the module is landing soon. Maybe this should hold and I can update the PR when that is published.

**Comment by lance** [comment]
@markusthoemmas new release of the module is landing soon. Maybe this should hold and I can update the PR when that is published.

**Comment by csantanapr** [comment]
Thanks for the ping @markusthoemmes 

@lance next time just open an issue I'm happy to update the sample or just assigned me a reviewer. 

I'll give the code a try. 

@lance how "soon" is the new update landing and how it changes your PR?

We can always merge this PR and do another when the updates to the SDK lands.

**Comment by lance** [comment]
@csantanapr there are 3.2.1 and 4.0.0 releases imminent - should land this week or early next at the latest. In 4.0.0 there are some [major interface changes](https://github.com/cloudevents/sdk-javascript/pull/324) that affect how a user receives CE messagaes. If you want to land this PR now, that's not a problem.

**Comment by csantanapr** [comment]
@lance oh I thought it was more like a 3.3 bump not a major API release. 

With 2.x 3.x and 4.x release so close together it looks like the SDK is having a lot of fun turns. 

I would preferred end users just ignore and not be aware of 3.x and focus on 4.x. The more API changes the less trust on the library.

**Comment by lance** [comment]
@csantanapr I went ahead and just bumped to 3.2.0. The 4.x version is still pending release.

**Comment by csantanapr** [comment]
I will review it @lance with 3.x then

File: docs/serving/samples/cloudevents/cloudevents-nodejs/index.js [hunk 1]
```diff
@@ -15,11 +15,10 @@ limitations under the License.
 */
 
 const express = require('express')
-const { CloudEvent, HTTPEmitter, HTTPReceiver } = require('cloudevents-sdk')
+const { CloudEvent, Emitter, HTTP, headersFor } = require('cloudevents')
```
Comment (csantanapr) [inline docs/serving/samples/cloudevents/cloudevents-nodejs/index.js:18]: Small nit `headersFor` is an unused variable

**Comment by csantanapr** [comment]
@lance it was a good thing base image has `arm64` cc @mattmoor

File: docs/serving/samples/cloudevents/cloudevents-nodejs/index.js (hunk 1 above)
Comment (lance) [reply docs/serving/samples/cloudevents/cloudevents-nodejs/index.js]: Fixed

**Pull request merged**